nbs/explains/minidataapi.ipynb

```diff
@@ -0,0 +1,889 @@
+{
```

Comment:
_Commented on notebook `nbs/explains/minidataapi.ipynb` Cell 66 Line 1_
```python
todos.xtra(name='Alma')
```
```todos.xtra(name='Charlie')``` I think is what you mean to have here